packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -75,12 +75,12 @@ Posts.addView("userPosts", terms => ({
   options: {
     limit: 5,
     sort: {
-      score: -1,
+      postedAt: -1,
     }
   }
 }));
 ensureIndex(Posts,
-  augmentForDefaultView({ userId: 1, score: -1 }),
+  augmentForDefaultView({ userId: 1, postedAt: -1 }),
   {
     name: "posts.userId_score",
```

Comment:
Also update the name to match:
```suggestion
    name: "posts.userId_postedAt",
```

The other thing here is making sure that the `posts.userId_score` index isn't also being used for a different view. I don't think it is, because `posts.userId_stickies_score` ends up covering it.